openapi-v3.yaml

```diff
@@ -86,7 +86,19 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/AccountsResponse'
-                
+          links:
+            GetAccountById:
+              operationId: getAccountById
+              parameters:
+                accountId: $response.body#/data/0/id
+            ListProjects:
+              operationId: listProjects
+              parameters:
+                accountId: $response.body#/data/0/id
+            ListJobsForAccount:
+              operationId: listJobsForAccount
+              parameters:
+                accountId: $response.body#/data/0/id
```

Comment:
Example of using links to enable API traversal

Reply:
FYI - super helpful, but I don't see anything come through when rendering via redoc

Reply:
Yeah, unfortunately Redocly nor Postman do not seem to do anything with this information at this time. However, other tools like [schemathesis can use this to generate a sequence of operations to test](https://schemathesis.readthedocs.io/en/stable/stateful.html).

Reply:
Ah, interesting!  Is the recommendation here that we replace our Redocly usage with Spectral?  Do we know all the places in which Redocly is currently used?

Reply:
Spectral will complement Redocly Reference, which is used for our API documentation. We won't be replacing Redocly